docs/DeveloperGuide.md

```diff
@@ -435,6 +435,18 @@ due to the presence of the special character `'` in the event name.
 We propose to allow the `createMember`, `editMember` , `createEvent` and `editEvent` commands to accept special 
 characters in the name field and not to be restricted to just alphanumeric characters.
 
+### Allow users to delete values from optional fields in member/event/enrolment
+**Current Implementation:**
+* **Current Issue:** Currently, there is no way for the user to delete unwanted values for optional fields in member/event/enrolment.
```

Comment:
Should we include that this problem does not apply to tags?
Something along the line of
```suggestion
* **Current Issue:** Currently, there is no way for the user to delete unwanted values for optional fields except for tags in member/event/enrolment.
```

Reply:
I think the original still applies, since we technically cannot delete the tags, but only overwrite them?

Reply:
I was thinking, from a user perspective, typing `t/` with a blank value essentially removes the tags, hence the idea that tags can be deleted. But your argument that tags are just being overwritten makes sense as well. This is just a philosophical question and I'm ok with leaving it as it was